recipes/cpuinfo/all/conanfile.py

```diff
@@ -72,6 +72,8 @@ def _configure_cmake(self):
         self._cmake.definitions["CLOG_RUNTIME_TYPE"] = "default"
         self._cmake.definitions["CLOG_BUILD_TESTS"] = False
         self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
+        if self.settings.arch == "armv8":
+            self._cmake.definitions["CMAKE_SYSTEM_PROCESSOR"] = "armv8"
```

Comment:
Is not set by conan? Seems to be a hack.

Reply:
for me, it wasn't set by conan. I doubt CMake helper (especially an old one) will ever set it... see https://github.com/conan-io/conan/pull/8026